rubygem-concurrent-ruby-edge/rubygem-concurrent-ruby-edge.spec

```diff
@@ -16,8 +16,7 @@ Source0: http://rubygems.org/gems/%{gem_name}-%{version}.gem
 
 Requires: %{?scl_prefix_ruby}ruby(rubygems)
 Requires: %{?scl_prefix_ruby}ruby
-Requires: %{?scl_prefix_ror}rubygem(concurrent-ruby) >= 1.0.0
-Requires: %{?scl_prefix_ror}rubygem(concurrent-ruby) < 1.1.0
+Requires: %{?scl_prefix_ror}rubygem(concurrent-ruby) = 1.0.3
```

Comment:
> it shouldn't be a problem for dynflow to build if foreman-packaging provides concurrent-ruby 1.0.

There is no foreman-packaging build of concurrent-ruby 1.0.3 for EL7 SCL nor Fedora 24. It only provides a _non-SCL EL7_ build at version 1.0.3, according to `comps/`.